README.md

```diff
@@ -1,6 +1,16 @@
-# chainlink
+![image](https://user-images.githubusercontent.com/75752467/126004352-d02952f7-fcb7-4afe-94ce-bc677f459bd9.png)
```

Comment:
You probably want to commit the image to the repo, then you can reference it by `/images/logo.png`. Bit of a personal choice, but depending on how you initially uploaded the image this might be more reliable.

The image is also a bit big. You could replace this with some html to specify a size:
```
<img src="/images/logo.png" alt="logo" width="200"/>
```
(Note: Markdown is converted to html, so you can always include html instead)